test/data/activeBandsEnWindow.in

```diff
@@ -0,0 +1,3 @@
+windowType = "energy" 
```

Comment:
Can we delete test/data/activeBands*Window.in?

Reply:
Right now I'm using those two files in the activeBandStructureTest file -- we could move them somewhere other than data, or I could try to figure out how to generate the test context without reading from input. 
https://github.com/mir-group/phoebe/blob/4896141a9c32d95cccf4a74f8dce62eae137c578/test/activeBandStructureTest.cpp#L24

Reply:
We could add a few methods to Context and write:
Context context1;
context.setWindowType("energy");
...
and then we can remove the two files.

Reply:
Yep, this works, I'll make the change.

Reply:
For Wednesday's meeting: I remember now why I didn't do this -- context doesn't have set functions for temperature, window type, energyRange, etc. We can discuss in person if we want me to add those functions or keep the input files elsewhere.

Reply:
Added those functions and removed the files

Reply:
Perfect, thank you.